### PR TITLE
technical updates to secrets backend setups

### PR DESCRIPTION
**Pull request by jwitz**
Part of https://github.com/astronomer/docs/issues/2921

Fixes the technical issues described in https://github.com/astronomer/docs/issues/2921#issuecomment-1758704914

### CODE REVIEW

File: astro/secrets-backend/azure-key-vault.md [hunk 1]
```diff
@@ -44,11 +44,11 @@ apache-airflow-providers-microsoft-azure
...
   
 ```text
-AZURE_CLIENT_ID="<your-client-id>" # Found on App Registration page > 'Application (Client) ID'
-AZURE_TENANT_ID="<your-tenant-id>" # Found on App Registration page > 'Directory (tenant) ID'
-AZURE_CLIENT_SECRET="<your-client-secret>" # Found on App Registration Page > Certificates and Secrets > Client Secrets > 'Value'
 AIRFLOW__SECRETS__BACKEND=airflow.providers.microsoft.azure.secrets.key_vault.AzureKeyVaultBackend
-AIRFLOW__SECRETS__BACKEND_KWARGS={"connections_prefix": "airflow-connections", "variables_prefix": "airflow-variables", "vault_url": "<your-vault-url>"}
+# Locate your client ID in Azure Portal under App Registration page > 'Application (Client) ID'
+# Locate your tenant ID in Azure Portal under App Registration page > 'Directory (tenant) ID'
+# Locate your client secret in Azure Portal under App Registration Page > Certificates and Secrets > Client Secrets > 'Value'
```
Comment (lzdanski) [inline astro/secrets-backend/azure-key-vault.md:50]: Screenshot 2023-12-22 at 11 46 41 AM
Is there a reason we have these in comments? It makes the code example a little clunky. they might be better as a bulleted list?

**Review by lzdanski — APPROVED** [review]
After tech review, LGTM! Just a small formatting note.

**Review by melugoyal — APPROVED** [review]
looks great, thank you!

File: astro/secrets-backend/azure-key-vault.md (hunk 1 above)
Comment (jwitz) [inline astro/secrets-backend/azure-key-vault.md:50]:
```suggestion
```

File: astro/secrets-backend/azure-key-vault.md [hunk 2]
```diff
@@ -44,11 +44,11 @@ apache-airflow-providers-microsoft-azure
...
+AIRFLOW__SECRETS__BACKEND_KWARGS={"connections_prefix": "airflow-connections", "variables_prefix": "airflow-variables", "vault_url": "<your-vault-url>", "tenant_id": "<your-tenant-id>", "client_id": "<your-client-id>", "client_secret": "<your-client-secret>"}
 ```
 
 This tells Airflow to look for variable information at the `airflow/variables/*` path in Azure Key Vault and connection information at the `airflow/connections/*` path. You can now run a DAG locally to check that your variables are accessible using `Variable.get("<your-variable-key>")`.
```
Comment (jwitz) [inline astro/secrets-backend/azure-key-vault.md:54]:
```suggestion
This configuration tells Airflow to look for variable information at the `airflow/variables/*` path in Azure Key Vault and connection information at the `airflow/connections/*` path. You can now run a DAG locally to check that your variables are accessible using `Variable.get("<your-variable-key>")`.
```

File: astro/secrets-backend/azure-key-vault.md [hunk 3]
```diff
@@ -44,11 +44,11 @@ apache-airflow-providers-microsoft-azure
...
-AZURE_CLIENT_ID="<your-client-id>" # Found on App Registration page > 'Application (Client) ID'
-AZURE_TENANT_ID="<your-tenant-id>" # Found on App Registration page > 'Directory (tenant) ID'
-AZURE_CLIENT_SECRET="<your-client-secret>" # Found on App Registration Page > Certificates and Secrets > Client Secrets > 'Value'
 AIRFLOW__SECRETS__BACKEND=airflow.providers.microsoft.azure.secrets.key_vault.AzureKeyVaultBackend
-AIRFLOW__SECRETS__BACKEND_KWARGS={"connections_prefix": "airflow-connections", "variables_prefix": "airflow-variables", "vault_url": "<your-vault-url>"}
+# Locate your client ID in Azure Portal under App Registration page > 'Application (Client) ID'
+# Locate your tenant ID in Azure Portal under App Registration page > 'Directory (tenant) ID'
+# Locate your client secret in Azure Portal under App Registration Page > Certificates and Secrets > Client Secrets > 'Value'
+AIRFLOW__SECRETS__BACKEND_KWARGS={"connections_prefix": "airflow-connections", "variables_prefix": "airflow-variables", "vault_url": "<your-vault-url>", "tenant_id": "<your-tenant-id>", "client_id": "<your-client-id>", "client_secret": "<your-client-secret>"}
 ```
```
Comment (jwitz) [inline astro/secrets-backend/azure-key-vault.md:53]:
```suggestion

To find your your client ID in Azure Portal, go to **App Registration page** > **Application (Client) ID**. To find your tenant ID, go to **App Registration page** > **Directory (tenant) ID**. To locate your client secret in Azure Portal, go to **App Registration Page** > **Certificates and Secrets** > **Client Secrets** > **Value**.

```

**Pull request merged**